extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/CriterionLiteralDto.java

```diff
@@ -0,0 +1,33 @@
+package de.sovity.edc.ext.wrapper.api.ui.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.List;
+
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@RequiredArgsConstructor
+@Schema(description = "Criterion Literal DTO")
+public class CriterionLiteralDto {
+
+    @Schema(description = "")
+    private String value;
+
+    @Schema(description = "")
```

Comment:
documentation should explain e.g.. "Only for type VALUE_LIST. List of values, e.g. for the IN-Operator`